js/turtleactions/ToneActions.js

```diff
@@ -16,6 +16,10 @@
  * MA 02110-1335 USA.
 */
 
+/*exported setupToneActions*/
+
+/* global Singer, logo, blocks, MusicBlocks, Mouse, last, _, VOICENAMES, instrumentsEffects, NOINPUTERRORMSG, arg */
```

Comment:
Same here for `arg`

Reply:
Yes, it is a global, I can be wrong but in this case arg is a global present in js/blocks sub-directory in multiple files in it. 

In this file, i.e. `js/turtleactions/ToneActions.js`  it is used inside the method for:- Frequency Modulator used to define a timbre.

Reply:
Not a global. This is an error I mistakenly introduced while porting to `turtleactions`. See f24a94bda391539f997308ad87b30d09c30f0f86. Fix it pls.